### PR TITLE
fix: call-management permission not doing anything

### PR DESCRIPTION
**Pull request by pierre-lehnen-rc**
## Issue(s)

SUP-211

### CODE REVIEW

File: apps/meteor/app/authorization/server/constant/permissions.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const permissions = [
 	{ _id: 'preview-c-room', roles: ['admin', 'user', 'anonymous'] },
 	{ _id: 'view-outside-room', roles: ['admin', 'owner', 'moderator', 'user'] },
 	{ _id: 'view-broadcast-member-list', roles: ['admin', 'owner', 'moderator'] },
-	{ _id: 'call-management', roles: ['admin', 'owner', 'moderator'] },
+	{ _id: 'call-management', roles: ['admin', 'owner', 'moderator', 'user'] },
```
Comment (debdutdeb) [inline apps/meteor/app/authorization/server/constant/permissions.ts:86]: should there not be a migration for this?

Comment (pierre-lehnen-rc) [reply apps/meteor/app/authorization/server/constant/permissions.ts:86]: There's already a migration calling upsertPermissions in there. If this PR gets left out of the release I'll include a new one.

File: apps/meteor/app/videobridge/client/tabBar.tsx [hunk 2]
```diff
@@ -57,7 +58,7 @@ addAction('start-call', ({ room }) => {
 	const live = room?.streamingOptions && room.streamingOptions.type === 'call';
 	const enabled = enabledDMs || enabledChannel || enabledTeams || enabledGroups || enabledLiveChat;
 
-	const enableOption = enabled && (!user?.username || !room.muted?.includes(user.username));
+	const enableOption = enabled && (!user?.username || !room.muted?.includes(user.username)) && canStartCall;
```
Comment (debdutdeb) [inline apps/meteor/app/videobridge/client/tabBar.tsx:61]: nit: can avoid some computation this way
```suggestion
	const enableOption = enabled && canStartCall && (!user?.username || !room.muted?.includes(user.username));
```

**Review by debdutdeb — APPROVED** [review]

**Review by debdutdeb — APPROVED** [review]

File: apps/meteor/app/api/server/v1/videoConference.ts [hunk 3]
```diff
@@ -26,6 +26,10 @@ API.v1.addRoute(
 				return API.v1.failure('invalid-params');
 			}
 
+			if (!(await hasPermissionAsync(userId, 'call-management', roomId))) {
+				return API.v1.failure('unauthorized');
```
Comment (sampaiodiego) [inline apps/meteor/app/api/server/v1/videoConference.ts:30]:
```suggestion
				return API.v1.unauthorized();
```

File: .changeset/call-management-permission.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': minor
```
Comment (sampaiodiego) [inline .changeset/call-management-permission.md:2]: 😬 
```suggestion
'@rocket.chat/meteor': patch
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**